drop_cap/demo/src/main/java/com/mecharyry/dropcap/FontType.java

```diff
@@ -0,0 +1,32 @@
+package com.mecharyry.dropcap;
```

Comment:
would having `demo` in the package name be less confusing (i.e. what is the library and what is the demo)